### ISSUE TITLE
MD5 returned by SQS does not match the calculation on the original request (ruby aws-sdk-core >= 2.3)

### ISSUE THREAD

**Issue by isacaraujo** [iain/fake_sqs#40]
Hi srs,

I've worked in a project with the ruby gem [Shoryuken](/phstc/shoryuken), and I got a problem:

It always worked well until I update the gem by `bundle update shoryuken`, updating the `aws-sdk-core` too.
Since that, I got the error:

```
Aws::Errors::ChecksumError:
       MD5 returned by SQS does not match the calculation on the original request. (MD5 calculated by the message atributes: 'ff41d67aace8f6c385e8a5071b828b5c', MD5 checksum returned: '')
```

The error occurs just version >= 2.3 of aws-sdk. I had no problems with versions up to that.

Looking the backtrace, I found this validation:
https://github.com/aws/aws-sdk-ruby/blob/v2.3.2/aws-sdk-core/lib/aws-sdk-core/plugins/sqs_md5s.rb#L75

**Comment by isacaraujo** [comment]
Sorry, the same problem of ticket https://github.com/iain/fake_sqs/issues/35

**Comment by algorist** [comment]
For anyone else looking into this problem, the fix for me (aws-sdk-core 2.7.3) was:
```
Aws.config[ :verify_checksums ] = false
```

**Comment by tiwilliam** [comment]
Duplicate of #35.
